chapter_1/Functions/functions(1).py

```diff
@@ -0,0 +1,26 @@
+import socket
+
+
+def ret_banner(ip, port):
+    try:
+        socket.setdefaulttimeout(2)
+        s = socket.socket()
+        s.connect((ip, port))
+        banner = s.recv(1024)
+        return banner
+    except:
+        return
+
+
+def main():
+    ip1 = '192.168.95.148'
+    ip2 = '192.168.95.149'
+    port = '21'
+    banner1 = ret_banner(ip1, port)
+    if banner1:
+        print('{+} ' + ip1 + ': ' + banner1)
+    banner2 = ret_banner(ip2, port)
+    if banner2:
+        print('{+}' + ip2 + ': ' + banner2)
+        if __name__ == '__main__':
```

Comment:
The indentation of line 25 & 26 is not correct, so this program will not run. Please correct.

```suggestion
if __name__ == '__main__':
```

Reply:
Done.